app/views/topics/_topic.html.erb

```diff
@@ -19,7 +19,14 @@
     </td>
   <% end %>
   <td>
-    <span class="more-important truncate-ellipsis"><%= link_to h(topic.name), topic_posts_path(topic) -%></span>
+    <span class="more-important truncate-ellipsis">
+      <a href="<%= topic_posts_path(topic) %>">
```

Comment:
Any particular reason for favouring this over 
```
<%= link_to topic_posts_path(topic) do %>
...
<% end %>
```

Reply:
i didnt know we can wrap html elements in link_top helpers like that. cool, i'll change that.